src/storage/storage-cache.ts

```diff
@@ -210,62 +210,15 @@ export class StorageCache {
         docs,
         expires: Date.now() + this._timeToLive,
       });
-      logger.debug("👹");
+      
       this._fireOnCacheUpdateds();
     });
 
     // Return an empty result for the moment.
     return [];
   }
 
-  // SET
```

Comment:
I should leave a `set` method that proxies on to the standard storage's set instead.